src/mango/src/mango_httpd.erl

```diff
@@ -266,6 +267,11 @@ run_find(Resp, Db, Sel, Opts) ->
     mango_crud:find(Db, Sel, fun handle_doc/2, Acc0, Opts).
 
 
+handle_doc({add_key, Key, Value = #execution_stats{}}, Acc0) ->
```

Comment:
It's not clear to me which code is redundant here. The intent is to perform custom JSON serialisation for the execution_stats record - without that, the response is invalid (JSON serialisation fails).